### PR TITLE
odb: log unsupported LEF properties

### PR DESCRIPTION
**Pull request by antonblanchard**
Don't silently ignore unsupported LEF properties.

### CODE REVIEW

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
You'll have a lot of tests to update

File: src/psm/test/aes_asap7_vdd.ok [hunk 1]
```diff
@@ -3,6 +3,10 @@
 The LEF parser will ignore this statement.
 To avoid this warning in the future, remove this statement from the LEF file with version 5.6 or later. See file asap7_data/asap7.lef at line 2.
 
+[INFO ODB-0388] unsupported LEF58_PITCH property for layer M2 :"
```
Comment (gadfort) [inline src/psm/test/aes_asap7_vdd.ok:6]: @maliberty  is this a parameter that's something we should support? It seems like it handles the asap7 tracks

Comment (maliberty) [reply src/psm/test/aes_asap7_vdd.ok:6]: Yes we should.

Comment (maliberty) [reply src/psm/test/aes_asap7_vdd.ok:6]: https://github.com/The-OpenROAD-Project/OpenROAD/issues/2926

**Review by maliberty — APPROVED** [review]

**Pull request merged**